### ISSUE TITLE
module 'arcade' has no attribute 'create_text'

### ISSUE THREAD

**Issue by lovelypython** [pythonarcade/arcade#1339]
![image](https://user-images.githubusercontent.com/96528749/192793089-f61a4560-96a2-4852-934b-e5244e99ddb1.png)
from arcade import *
Traceback (most recent call last):
  File "<pyshell#13>", line 1, in <module>
    from arcade import *
AttributeError: module 'arcade' has no attribute 'create_text'. Did you mean: 'create_line'?

Is this a bug for arcade?
And my version is the latest2.6.16
How to fix it?

**Comment by einarf** [comment]
`create_text` was deprecated and removed.

It's always a good idea to look through the release notes: https://api.arcade.academy/en/latest/development/release_notes.html

**Comment by Cleptomania** [comment]
This is a bug in Arcade, however I doubt we will fix it in a 2.6 release. It is already fixed in our upcoming 2.7 development branch.

In the meantime, you can use `import arcade` instead of `from arcade import *`. In general doing a full static import of a package is somewhat considered bad practice as you lose the ability to namespace functions/classes

@einarf `__init__.__all__` does still contain a reference to this, which is why it’s being triggered from the static import.

**Comment by einarf** [comment]
> @einarf `__init__.__all__` does still contain a reference to this, which is why it’s being triggered from the static import.

ah right. We should add some sanity check for that.

**Comment by pushfoo** [comment]
@lovelypython 
You can use `arcade.Text` objects instead of the old `create_text` method. It's advisable to use `Text` objects instead of `arcade.draw_text` because they render up to to 100 times faster. Here's a small example of how to use them:

```py
from itertools import cycle


import arcade
from arcade import Color, Window
from arcade.color import RED, BLUE, GREEN, WHITE


# This repeats the color choices forever
COLOR_CHOICES = cycle((RED, BLUE, GREEN))


class Example(Window):

    def __init__(self):
        super().__init__(800, 600, "Click to add Text objects!")
        self.text_objects = []
        self.text_spawn_x = None
        self.text_spawn_y = None
        self.setup()

    def add_text_object(self, text: str, color: Color = WHITE):
        text_object = arcade.Text(
            text, self.text_spawn_x - 80, self.text_spawn_y, color=color)
        self.text_objects.append(text_object)
        self.text_spawn_y -= 20

    def setup(self):
        self.text_objects.clear()
        self.text_spawn_x = self.width // 2
        self.text_spawn_y = self.height - 20
        self.add_text_object("Click to add text objects!")

    def on_mouse_release(self, x: int, y: int, button: int, modifiers: int):
        # Add a new text object whenever the mouse is released
        self.add_text_object(f"Clicked {len(self.text_objects) - 1} times!", next(COLOR_CHOICES))

    def on_draw(self):
        arcade.start_render()
        for text in self.text_objects:
            text.draw()


if __name__ == "__main__":
    example = Example()
    arcade.run()
```
For more information, you can check the following docs:
* [Drawing Text Objects](https://api.arcade.academy/en/stable/examples/drawing_text_objects.html). 
* [arcade.Text in the API reference](https://api.arcade.academy/en/latest/api/text_pyglet.html#arcade-text)

**Comment by einarf** [comment]
> they render up to to 100 times faster

More like 100-1000 times faster, but it stresses the reder queue if you have too many and you'll need the 2.7 batching.

**Comment by einarf** [comment]
Discussed on Discord: We should at the very least add some tests for this so it doesn't happen again in the future.

**Comment by albertz** [comment]
Could you just replace `create_text` by `CreateText` and it would work?

**Edit** Sorry, this was with Arcade 2.3. It seems this is also gone now. I guess now it's just `Text`, and the `render_text` would be replaced by `label.draw()`.

**Comment by einarf** [comment]
Added test : https://github.com/pythonarcade/arcade/pull/1412

**Comment by einarf** [comment]
It was fixed in 2.6 branch and we added unit tsts. Closing
